LockboxXCUITests/LockboxXCUITests.swift

```diff
@@ -197,6 +186,13 @@ class LockboxXCUITests: BaseTestCase {
         waitforExistence(app.tables.cells.staticTexts["iosmztest@gmail.com"])
     }
 
+    func iPad() -> Bool {
```

Comment:
This is the idea, thanks! bug instead of creating this in the test case, can we do it in the [BaseTestcase](https://github.com/mozilla-lockbox/lockbox-ios/blob/master/LockboxXCUITests/BaseTestCase.swift) file instad?